CONTRIBUTING.md

```diff
@@ -0,0 +1,159 @@
+# Contributing
+
+When contributing to this repository, please first discuss the change you wish
+to make via issue, email, or any other method with the owners of this repository
+before making a change.
+
+Please note we have a code of conduct, please follow it in all your interactions
+with the project.
+
+## Intellectual Property Rights
+
+All contributions are only accepted under the license in the accompanying
+LICENSE.md file.  This means that when you make a contribution, you are agreeing
```

Comment:
I believe that the license file is just named `LICENSE` not `LICENSE.md`.

Reply:
Whoops!  I haven't checked, did you make the change?